codec-haproxy/src/main/java/io/netty/handler/codec/haproxy/HAProxyMessage.java

```diff
@@ -182,26 +202,28 @@ static HAProxyMessage decodeHeader(ByteBuf header) {
                 addressLen = addressEnd - startIdx;
             }
             dstAddress = header.toString(startIdx, addressLen, CharsetUtil.US_ASCII);
+            //AF_UNIX defines that excactly 108 bytes are reserved for the address
```

Comment:
@dobermai this sounds like a bug then... Can you open another PR where you just fix this ?